app/helpers/members_helper.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
-# MembersHelper stub
+# Member helper
 module MembersHelper
+  def search_params
```

Comment:
Not a blocker, but a friendly suggestion that `search_query` might be a more consistent name for this method.